src/current/molt/molt-fetch.md

```diff
@@ -70,37 +60,64 @@ Complete the following items before using MOLT Fetch:
+
+- [Replicate changes](#replication-only)
+- [Resume replication](#resume-replication)
+
 ## Best practices
 
 {{site.data.alerts.callout_success}}
 To verify that your connections and configuration work properly, run MOLT Fetch in a staging environment before migrating any data in production. Use a test or development environment that closely resembles production.
 {{site.data.alerts.end}}
 
-- To prevent connections from terminating prematurely during data export, set the following to high values on the source database:
+- To prevent connections from terminating prematurely during the [data export phase](#data-export-phase), set the following to high values on the source database:
 
 	- **Maximum allowed number of connections.** MOLT Fetch can export data across multiple connections. The number of connections it will create is the number of shards ([`--export-concurrency`](#global-flags)) multiplied by the number of tables ([`--table-concurrency`](#global-flags)) being exported concurrently.
 
 		{{site.data.alerts.callout_info}}
-		Only tables with [primary key]({% link {{ site.current_cloud_version }}/primary-key.md %}) types of [`INT`]({% link {{ site.current_cloud_version }}/int.md %}), [`FLOAT`]({% link {{ site.current_cloud_version }}/float.md %}), or [`UUID`]({% link {{ site.current_cloud_version }}/uuid.md %}) can be sharded.
+		With the default numerical range sharding, only tables with [primary key]({% link {{ site.current_cloud_version }}/primary-key.md %}) types of [`INT`]({% link {{ site.current_cloud_version }}/int.md %}), [`FLOAT`]({% link {{ site.current_cloud_version }}/float.md %}), or [`UUID`]({% link {{ site.current_cloud_version }}/uuid.md %}) can be sharded. PostgreSQL users can enable [`--use-stats-based-sharding`](#global-flags) to use statistics-based sharding for tables with primary keys of any data type. For details, refer to [Table sharding](#table-sharding).
 		{{site.data.alerts.end}}
 
 	- **Maximum lifetime of a connection.**
 
-- If a PostgreSQL database is set as a [source](#source-and-target-databases), ensure that [`idle_in_transaction_session_timeout`](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-IDLE-IN-TRANSACTION-SESSION-TIMEOUT) on PostgreSQL is either disabled or set to a value longer than the duration of data export. Otherwise, the connection will be prematurely terminated. To estimate the time needed to export the PostgreSQL tables, you can perform a dry run and sum the value of [`molt_fetch_table_export_duration_ms`](#metrics) for all exported tables.
+- For PostgreSQL sources using [`--use-stats-based-sharding`](#global-flags), run [`ANALYZE`]({% link {{ site.current_cloud_version }}/create-statistics.md %}) on source tables before migration to ensure optimal shard distribution. This is especially important for large tables where even distribution can significantly improve export performance.
```

Comment:
I think it can be called out here that manually running Analyze is fine, but most likely users have auto vacuum on which would  collect these stats automatically. Running the analyze manually just ensures all stats are up to date, but this is not necessary for this to work unless there are no statistics collected yet which would be unlikely for a production table.

Reply:
Oh nevermind i see it also covered in the section below now. This should be good then.